packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -995,7 +1010,7 @@ class _NavigationBarDestinationSemantics extends StatelessWidget {
       builder: (BuildContext context, Widget? child) {
         return Semantics(
           selected: destinationInfo.selectedAnimation.isForwardOrCompleted,
-          container: true,
```

Comment:
Navigationbar now will have explicitchildnode so this is no longer needed, this also allowing this semantics to merge up to the semantics role